src/client.ts

```diff
@@ -9974,6 +9963,14 @@ export class MatrixClient extends TypedEventEmitter<EmittedEvents, ClientEventHa
             throw err;
         }
     }
+
+    public async getAuthIssuer(): Promise<{
+        issuer: string;
+    }> {
```

Comment:
I can't say I'm a huge fan of adding even more methods to `MatrixClient`, but I suppose that's a discussion for another day. If you're going to do it, document it and give it test coverage, please.